Libraries/LibJS/Runtime/Object.h

```diff
@@ -169,7 +169,7 @@ class Object : public Cell {
 
     bool m_is_extensible { true };
     Shape* m_shape { nullptr };
-    Vector<Value> m_storage;
+    MarkedValueList m_storage;
```

Comment:
This change is not needed, as `JS::Object` will mark the values in its storage in `Object::visit_children()`

Reply:
Should I make `Object` _not_ mark these, or just revert this (and the one below)?

Reply:
Just revert it :)
We have to visit the objects anyway, so let's avoid tracking more MVL's